src/transformers/models/gpt2/modeling_gpt2.py

```diff
@@ -297,6 +318,7 @@ def forward(
         encoder_attention_mask: Optional[torch.FloatTensor] = None,
         use_cache: Optional[bool] = False,
         output_attentions: Optional[bool] = False,
+        padding_mask: Optional[torch.Tensor] = None,
```

Comment:
We now deprecated `padding_mask` in favor of `attention_mask`, can you follow a similar logic than: https://github.com/huggingface/transformers/blob/main/src/transformers/models/falcon/modeling_falcon.py#L1028-L1035 and pass `attention_mask` instead of `padding_mask`

Reply:
done